src/hooks/useVestings.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { DclData } from '../clients/DclData'
+
+import { DEFAULT_QUERY_STALE_TIME } from './constants'
+
+function useVestings(shouldFetch: boolean) {
+  const { data, isLoading: isLoadingVestingsData } = useQuery({
+    queryKey: ['vestings'],
+    queryFn: () => DclData.get().getVestings(),
+    staleTime: DEFAULT_QUERY_STALE_TIME,
```

Comment:
I think this stale time could be 24hs, if the exported data from transparency only gets updated daily